docker/cvmfs-install/Readme.md

```diff
@@ -0,0 +1,127 @@
+
+
+# CVMFS installation with docker images
+
+CVMFS is one of the main deployment solution for the iLCSoft stack. CVMFS provides servers to software teams to install and publish their packages. 
+Setting up such servers requires some effort and are not always available for all OS flavors that are supported in the different labs. 
+
+Here at DESY, the main OSs that are supported are SL6 (end of life in November 2020), Centos7 and Ubuntu 18.04. Only one CVMFS server is available for our deployment. To be able to install the iLCSoft stack on all possible OS flavors, we provide Docker images for these OS flavors within which the iLCSoft stack is compiled and then deployed later on on our CVMFS server. 
+
+As of today, the available images are:
+- centos7: `ilcsoft/ilcsoft-cvmfs-install-centos7`
+
+## Generating a docker image
+
+The Dockerfiles available in this directory are meant for updating or re-generating images for a CVMFS deployment.
+These images are generally already available on Dockerhub, so this step is not required. 
+
+If you need to create or re-generate an image, simply pickup a Dockerfile and e.g run:
+
+```shell
+# Build an image
+docker build -f Dockerfile.cvmfs-install.centos7 --tag ilcsoft/ilcsoft-cvmfs-install-centos7 .
+# Push the image on Dockerhub, if you have the permission dude!
+docker push ilcsoft/ilcsoft-cvmfs-install-centos7
+```
+
+## Compiling the stack
+
+For example here, we assume that you are using the centos7 image. The installation must performed on a machine where the stack can be uploaded later on to the CVMFS server (using `rsync`, see below). At DESY, any NAF machine is suitable. Please, also ensure you have enough disk space before starting the installation. Depending on the machine, docker might or not be available. The NAF machines at DESY currently do not support `docker` but have `singularity` installed.
```

Comment:
Regarding disk space are there ways to get a rough estimate for it before starting the whole process? Or some ball park estimates that can be used here from past experience?

Reply:
Maybe between 10 and 20 Gb. Before the installation is finished the build directories take a lot of space. After deletion, around 10 Gb I think.